x-pack/test/apm_api_integration/basic/tests/transaction_groups/breakdown.ts

```diff
@@ -33,22 +33,35 @@ export default function ApiTest({ getService }: FtrProviderContext) {
       before(() => esArchiver.load('8.0.0'));
       after(() => esArchiver.unload('8.0.0'));
 
-      it('returns the transaction breakdown', async () => {
+      it('returns the transaction breakdown for a service', async () => {
         const response = await supertest.get(
           `/api/apm/services/opbeans-node/transaction_groups/breakdown?start=${start}&end=${end}&uiFilters=${uiFilters}&transactionType=${transactionType}`
         );
 
         expect(response.status).to.be(200);
         expect(response.body).to.eql(expectedBreakdown);
       });
-      it('returns the transaction breakdown with transaction name', async () => {
+      it('returns the transaction breakdown for a transaction group', async () => {
         const response = await supertest.get(
           `/api/apm/services/opbeans-node/transaction_groups/breakdown?start=${start}&end=${end}&uiFilters=${uiFilters}&transactionType=${transactionType}&transactionName=${transactionName}`
         );
 
         expect(response.status).to.be(200);
         expect(response.body).to.eql(expectedBreakdownWithTransactionName);
       });
+      it('returns the kpis sorted by percentage and name', async () => {
+        const response = await supertest.get(
+          `/api/apm/services/opbeans-node/transaction_groups/breakdown?start=${start}&end=${end}&uiFilters=${uiFilters}&transactionType=${transactionType}`
+        );
+
+        expect(response.status).to.be(200);
+        expect(response.body.kpis).to.eql([
+          { name: 'app', percentage: 0.16700861715223636, color: '#54b399' },
+          { name: 'http', percentage: 0.7702092736971686, color: '#6092c0' },
+          { name: 'postgresql', percentage: 0.0508822322527698, color: '#d36086' },
+          { name: 'redis', percentage: 0.011899876897825195, color: '#9170b8' },
+        ]);
```

Comment:
```suggestion
        expect(response.body.kpis.map(kpi => kpi.name)).to.eql(["app", "http", "postgresql", "redis"]);
```

Reply:
This will make the test less error prone if the data changes, but it still catches regressions to the important logic. I actually think this test is much better than the generic tests we already have.